src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2115,42 +2219,60 @@
+          text: |
+            Vm1Id=$(az vm show --resource-group MyResourceGroup--name MyVm1 --query id --out tsv) \\
+                \\n\\n Vm2Id=$(az vm show --resource-group MyResourceGroup--name MyVm2 --query id --out tsv) \\
+                \\n\\n az network watcher test-connectivity --source-resource $MyVmId1 --dest-resource $MyVmId2 --dest-port 80
 """
 
 helps['network watcher show-next-hop'] = """
     type: command
     short-summary: Get information on the 'next hop' for a VM.
-    long-summary: Requires that Network Watcher is enabled for the region in which the VM is located.
+    long-summary: Requires that Network Watcher is enabled for the region in which the VM is located. For more information about show-next-hop visit https://docs.microsoft.com/en-us/azure/network-watcher/network-watcher-check-next-hop-cli
+    examples:
+        - name: Get the next hop from a VM's assigned IP address to a destination at 10.1.0.4.
+          text: az network watcher show-next-hop -g MyResourceGroup --vm MyVm --source-ip 10.0.0.4 --dest-ip 10.1.0.4
 """
 
 helps['network watcher show-security-group-view'] = """
     type: command
     short-summary: Get detailed security information on a VM for the currently configured network security group.
+    long-summary: For more information about security-group-view visit https://docs.microsoft.com/en-us/azure/network-watcher/network-watcher-security-group-view-cli
+    examples:
+        - name: Get the network security group information for the specified VM.
+          text: az network watcher show-security-group-view -g MyResourceGroup --vm MyVm
 """
 
 helps['network watcher show-topology'] = """
     type: command
     short-summary: Get the network topology of a resource group.
+    long-summary: For more information about show-topology visit https://docs.microsoft.com/en-us/azure/network-watcher/network-watcher-topology-cli
     parameters:
         - name: --resource-group -g
           short-summary: The name of the target resource group to perform topology on.
         - name: --location -l
           short-summary: Location. Defaults to the location of the target resource group.
-          long-summary: Topology information is only shown for resources within the target
+          long-summary: >
+            Topology information is only shown for resources within the target
             resource group that are within the specified region.
 """
 
 helps['network watcher packet-capture'] = """
     type: group
     short-summary: Manage packet capture sessions on VMs.
-    long-summary: |
-        These commands require that both Azure Network Watcher is enabled for the
-        VM's region and that AzureNetworkWatcherExtension is enabled on the VM.
+    long-summary: >
+        These commands require that both Azure Network Watcher is enabled for the VM's region and that AzureNetworkWatcherExtension is enabled on the VM. 
```

Comment:
This precondition listed in the `long-summary` should be a part of the `long-summary` for all `packet-capture` commands.

Reply:
Fixed in my branch.